File: C++/Searching/linear_search.cpp
#include <bits/stdc++.h>

using namespace std;

/*
<----------------------------------PROBLEM------------------------------------>
Given an array arr[] of n elements, write a function to search a given element el in arr[].

Variables used:
n = total number of elements in array
v = vector (dynamic array initialized with the input size)
el = element to be searched in the array. 
*/

int main()
{
    int n;
    cin >> n;
    vector<int> v(n);
    for(int i=0; i<n; i++){
        cin >> v[i];
    }
    
    int el;
    cin >> el;

    // Flag variable to keep a check if the element is found till then.
    bool flag=false;
    for(int i=0; i<n; i++){
        if(v[i]==el){
            flag=true;
        }
    }
    if(flag){
        cout << "Element found:)";
    }
    else{
        cout << "Element not found:(\n";
    }
    return 0;
}

File: Python/Searching/linear_search.py
'''
<----------------------------------PROBLEM------------------------------------>
Given an array arr[] of n elements, write a function to search a given element el in arr[].

Variables used:
n = total number of elements in array
arr = list of elements
el = element to be searched in the array.
'''

n = int(input())
arr = []
for i in range(n):
    x = int(input())
    arr.append(x)
el = int(input())
# Flag variable to keep a check if the element is found till then.
flag = False
for i in arr:
    if i == el:
        flag = True

if flag:
    print("Element found:)\n")
else:
    print("Element not found:(\n")